docs/gpa.md

```diff
@@ -233,78 +248,100 @@ The information included in the notice of intended procurement covers all the in
         <td markdown=1>
 
-* 
+* Please follow the guidance for VII:(k)
 </td>
       </tr>
       <tr>
         <td>X:7(d)</td>
         <td>where the procuring entity will conduct the procurement by electronic means, any authentication and encryption requirements or other requirements related to the submission of information by electronic means;</td>
         <td markdown=1>
 
-* 
+* Set `tender/submissionMethod` to 'electronicSubmission'
+* Enter or append *authentication and encryption requirements or other requirements related to the submission of information by electronic means* in `tender/submissionMethodDetails`)
+* If the electronic communication with the procuring entity requires the use of tools and devices that are not generally available, enter the Web address of these tools in `tender.communication.atypicalToolUrl`
+
+This requires the [Communication](https://github.com/open-contracting-extensions/ocds_communication_extension) extension.
 </td>
       </tr>
       <tr>
         <td>X:7(e)</td>
         <td>where the procuring entity will hold an electronic auction, the rules, including identification of the elements of the tender related to the evaluation criteria, on which the auction will be conducted;</td>
         <td markdown=1>
 
-* 
+* Set `tender/techniques/hasElectronicAuction` to `true`
+* Enter *the rules, including identification of the elements of the tender related to the evaluation criteria, on which the auction will be conducted* in `tender/techniques/electronicAuction/description`
+
+This requires the [Techniques](https://github.com/open-contracting-extensions/ocds_techniques_extension) extension.
 </td>
       </tr>
       <tr>
         <td>X:7(f)</td>
         <td>where there will be a public opening of tenders, the date, time and place for the opening and, where appropriate, the persons authorized to be present;</td>
         <td markdown=1>
 
-* 
+* Enter the *date* and the *time* in `tender/bidOpening/date`
+* Enter the *place* in `tender/bidOpening/address`, `tender/bidOpening/location` or `tender/bidOpening/gazetteer`
+* Enter *the persons authorized to be present* in `tender/bidOpening/description`
+
+This requires the [Bid Opening](https://github.com/open-contracting-extensions/ocds_bidOpening_extension) extension.
 </td>
       </tr>
       <tr>
         <td>X:7(g)</td>
         <td>any other terms or conditions, including terms of payment and any limitation on the means by which tenders may be submitted, such as whether on paper or by electronic means; and</td>
         <td markdown=1>
 
-* 
+* Enter the *terms of payment* in `tender/participationFees`
+
+This requires the [Participation Fees](https://github.com/open-contracting-extensions/ocds_participationFees_extension) extension.
+
+* Enter *the means by which tenders may be submitted* in `tender/submissionMethod`
+  * If it's on *paper*, enter 'written'
+  * If it's *electronic*, enter 'electronicSubmission'
```

Comment:
```suggestion
* Enter *the means by which tenders may be submitted* in `tender/submissionMethod`:
  * If it's *on paper*, enter 'written'
  * If it's *by electronic means*, enter 'electronicSubmission'
```

Reply:
Done in e2534dbe1e9

Reply:
Thanks! By the way, you can click "Add suggestion to batch" or "Commit suggestion" to commit multiple suggested changes in one commit.